python/src/fast.cpp

```diff
@@ -278,7 +278,9 @@ void init_fast(nb::module_& parent_module) {
               output_shapes (List[Sequence[int]]): The list of shapes for each output in ``output_names``.
               output_dtypes (List[Dtype]): The list of data types for each output in ``output_names``.
               grid (tuple[int, int, int]): 3-tuple specifying the grid to launch the kernel with.
+                This will be passed to MTLComputeCommandEncoder::dispatch_threads.
               threadgroup (tuple[int, int, int]): 3-tuple specifying the threadgroup size to use.
+                This will be passed to MTLComputeCommandEncoder::dispatch_threads.
```

Comment:
Nit:

```suggestion
                This will be passed to ``MTLComputeCommandEncoder::dispatch_threads``.
              threadgroup (tuple[int, int, int]): 3-tuple specifying the threadgroup size to use.
                This will be passed to ``MTLComputeCommandEncoder::dispatch_threads``.
```